### PR TITLE
Using ES6 article

### PR DESCRIPTION
**Pull request by ebidel**
R @arthurevans, @kaycebasques

### CODE REVIEW

File: 1.0/articles/es6.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+      ...
+    }
+
+    document.registerElement('stock-ticker', StockTicker);
+
+There's nothing new here. We're used to registering custom elements by passing
+`document.registerElement` the element name followed by a `prototype`. The only difference
+with classes is that you provide a `class` rather than a `prototype`.
+
+**Classes are prototypes!** Strictly speaking, classes are syntactic sugar for `prototype`s. You may not see the `prototype` keyword when creating elements from ES6 classes, but in actuality, a `protototype` is still being passed under the hood. The JS engine does this for you.
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: typo: `protototype`

File: 1.0/articles/es6.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+  <paper-button raised class="blue">Stock Ticker Demo</paper-button>
+</a>
+<a href="https://github.com/ebidel/polymer-experiments/tree/master/polymersummit/stockticker" target="_blank">
+  <paper-button raised class="blue">Full source</paper-button>
+</a>
+</p>
+
+## Building for production
+
+**Note** ES6 classes currently work in Chrome 42+, Safari 9+, Edge preview, and FF nightly.
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: Move to top of article?

File: 1.0/articles/es6.md (hunk 1 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Done

File: 1.0/articles/es6.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
+---
+layout: default
+type: article
+shortname: Articles
+title: "Building web components using ES6 classes"
+#subtitle: How to use classes and other new ES2015 features with Polymer.
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: delete?

File: 1.0/articles/es6.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+article: true
+description: How to build vanilla custom elements and Polymer elements using ES2015 features.
+published: 2015-09-24
+#updated: 2015-09-22
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: delete?

File: 1.0/articles/es6.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+      // Use createdCallback instead of constructor to init an element.
+      createdCallback() {
+        // This element uses Shadow DOM.
+        this.createShadowRoot().innerHTML = `
+          <style>
+            :host {
+              display: block;
+            }
+          </style>
+         <div id="quotes"><div>
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: indent one space?

File: 1.0/articles/es6.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+
+**Note** ES6 classes currently work in Chrome 42+, Safari 9+, Edge preview, and FF nightly.
+{: .alert .alert-info }
+
+For our ES6 code to work in all modern browsers, we need to transpile it to an ES5 equivalent. [BabelJS](https://babeljs.io/) is my personal favorite. It's the most popular at the moment and comes with
+a convenient CLI and Gulp/Grunt/Browserify workflows.
+
+To support all browsers, follow these steps:
+
+1. [Vulcanize](concatenating-web-components.html) your element(s) by inlining the script and CSS:
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: Can you check this link? I think it might need to be `../0.5/concatenating...`

File: 1.0/articles/es6.md [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+
+{% include toc.html %}
+
+## Introduction
+
+Web components evolve markup into something that's meaningful, maintainable, and highly modular. Thanks to these new API primitives, not only do we have improved ergonomics when building apps, but we gain better overall structure, design, and reusability.
+
+ES6 does the same thing, but for JavaScript. JavaScript is the core language of the web and it's about time it saw a makeover! ES6 brings powerful new primitives to the language, but they're mostly about developer convenience. The result of all the syntactical goodness? More legible, more maintainable, and more modular JavaScript.
+
+The analogy is simple: **Web Components : HTML :: ES6 : JS**. That is to say, what web components do for HTML, ES6 does for JavaScript
```
Comment (kaycebasques) [inline 1.0/articles/es6.md]: Add period to end of sentence.

**Comment by kaycebasques** [comment]
Built the site. Proofread. Checked links. Didn't test out the sample code, but the live sample site works. LGTM.

File: 1.0/articles/es6.md (hunk 3 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Done

File: 1.0/articles/es6.md (hunk 4 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Keeping in case this gets updated in the future. It will!

File: 1.0/articles/es6.md (hunk 5 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Done

File: 1.0/articles/es6.md (hunk 6 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Done

File: 1.0/articles/es6.md (hunk 7 above)
Comment (ebidel) [inline 1.0/articles/es6.md]: Done

File: 1.0/articles/es6.md [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+the `createdCallback` [custom element lifecycle method](../docs/devguide/registering-elements.html#lifecycle-callbacks) for setup work. Use it to initialize properties, (optionally) create [Shadow DOM](http://www.html5rocks.com/tutorials/webcomponents/shadowdom/) for the element, set default attributes, etc.
+
+The following example creates a class definition for our stocker ticker element:
+
+    class StockTicker extends HTMLElement {
+
+      // Use createdCallback instead of constructor to init an element.
+      createdCallback() {
+        // This element uses Shadow DOM.
+        this.createShadowRoot().innerHTML = `
```
Comment (arthurevans) [inline 1.0/articles/es6.md]: Is this the template string? You mention it below and it's super hard to spot in code if it's just a pair of backticks.

I'd suggest introducing the idea here, like // set HTML using a template string, explained below

At the risk of adding length, might even want to break this into two lines, so it's:

         // This element uses Shadow DOM.
         var shadowRoot = this.createShadowRoot();
          // Initialize shadow DOM using a template string (discussed below)
          shadowRoot.innerHTML = `

File: 1.0/articles/es6.md [hunk 9]
```diff
@@ -0,0 +1,298 @@
...
+{: .alert .alert-info }
+
+## Defining Polymer elements from a class
+
+Defining a Polymer element is similar to defining a vanilla custom element, with
+several important differences:
+
+1. There's no need to use `extends HTMLElement`. Polymer extends the element for you.
+2. Use the `beforeRegister` method instead of `createdCallback`. This is a special method
+Polymer calls before creating your element and is where you should define your `is` property (element's name) and [`properties` object](../docs/devguide/properties.html).
```
Comment (arthurevans) [inline 1.0/articles/es6.md]: What about listeners, observers, behaviors ...? Are those in the same boat?

File: 1.0/articles/es6.md [hunk 10]
```diff
@@ -0,0 +1,298 @@
...
+
+2. Pipe the output to [cripser](https://www.npmjs.com/package/gulp-crisper). Crisper is a tool that extracts all inline `<script>` and creates a standalone .js file.
+
+        crisper -s element.v.html -h element.v.html -j element.js
+
+3. Run the standalone JS file through Babel:
+
+        babel element.js -o element.js
+
+   **Note**: I'm overwriting the original source with Babel's output so we don't have to
```
Comment (arthurevans) [inline 1.0/articles/es6.md]: Maybe make clear that by "original" you mean, "the JavaScript source file created by crisper."

**Comment by arthurevans** [comment]
Added  a few retro-view comments. Otherwise LGTM. Ship it! (Oh, wait ...)

**Pull request merged**